src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -2,0 +1,31 @@
+import { FC } from 'react';
+import { Todo } from '../../api/todos';
+import { User } from '../../api/users';
+import { UserInfo } from '../UserInfo';
+import usersFromServer from '../../api/users';
+
+interface Props {
+  todo: Todo;
+  // user?: User;
+}
```

Comment:
The `Props` interface currently only includes the `todo` property. If you intend to pass user data as a prop, ensure that the `Props` interface is updated accordingly to include a `user` property or any other necessary properties.